### ISSUE TITLE
Dark mode for docs

### ISSUE THREAD

**Issue by illume** [pygame/pygame#3460]
Honoring the browser preference, with a button to toggle it, and a cookie to remember it.

**Comment by novialriptide** [comment]
I was thinking, is it a good idea to create a new documentation website? I'm particularly a big fan of [readthedocs.org's format](https://pytmx.readthedocs.io/en/latest/). We can set a toggle between the old and new docs just like https://old.reddit.com/

**Comment by illume** [comment]
This issue is about adding a dark mode, not about replacing the docs with a new website.

**Comment by itzpr3d4t0r** [comment]
I'm super excited for a dark mode, that green is historic but hurts the eyes a lot. With that being said i think we should show some examples before accepting one, as "dark mode" is too general.

**Comment by osinmv** [comment]
<img width="1680" alt="Screenshot 2022-10-26 at 3 10 13 PM" src="https://user-images.githubusercontent.com/31288523/198115711-51e1af1a-bb2f-414a-b22e-d61782d39fa3.png">

Does it look any good?

**Comment by novialriptide** [comment]
> <img alt="Screenshot 2022-10-26 at 3 10 13 PM" width="1680" src="https://user-images.githubusercontent.com/31288523/198115711-51e1af1a-bb2f-414a-b22e-d61782d39fa3.png">
> 
> Does it look any good? Just a concept

I feel that we would need to rewrite a lot of the CSS for a potential dark mode. The bright green text hurts my eyes a bit.

**Comment by osinmv** [comment]
Thanks, I will also talk with some people in pygame discord, maybe they will have some input on this. Once I gather info from them I will make a PR

**Comment by Zireael07** [comment]
Agreed, bright green on black while a throwback to the earliest terminals, makes my eyes hurt.

**Comment by itzpr3d4t0r** [comment]
In theory each category could be of a different color / color group, just to point out what each piece of text represents and guide the eye trough the site.
